src/tests/ftest/pool/RebuildTests.py

```diff
@@ -192,39 +169,27 @@ def test_simple_rebuild(self):
                 self.fail("Number of disabled targets reporting incorrectly: {}"
                           .format(pool.pool_info.pi_ndisabled))
             if pool.pool_info.pi_rebuild_st.rs_errno != 0:
-                self.fail("Rebuild error reported: {}".format(
-                    pool.pool_info.pi_rebuild_st.rs_errno))
-            if pool.pool_info.pi_rebuild_st.rs_obj_nr != objcount:
+                self.fail("Rebuild error reported: {}"
+                          .format(pool.pool_info.pi_rebuild_st.rs_errno))
+            if pool.pool_info.pi_rebuild_st.rs_obj_nr != self.objcount:
                 self.fail("Rebuilt objs not as expected: {0} {1}"
-                          .format(pool.pool_info.pi_rebuild_st.rs_obj_nr,
-                                  objcount))
-            if pool.pool_info.pi_rebuild_st.rs_rec_nr != (reccount*objcount):
+                          .format(pool.pool_info.pi_rebuild_st.rs_obj_nr, self.objcount))
```

Comment:
(pylint-line-too-long) Line too long (89/80)

Reply:
no belong to current patch.